app/views/admin/editions/locked.html.erb

```diff
@@ -0,0 +1,24 @@
+<% page_title @edition.title, @edition.format_name %>
+<% page_class "edition-show" %>
+<% initialise_script "GOVUK.adminEditionShow" %>
+
+<div class="row" data-module="track-button-click" data-track-category="button-pressed" data-track-action="<%= @edition.format_name.parameterize(separator: '-') %>-button">
```

Comment:
Do we know if this tracking logic still applies? I'm not sure which button it's supposed to track

Reply:
Similar to above, it wraps around the "show_metadata" partial in the show view (https://github.com/alphagov/whitehall/blob/374ee2ec2ed0964e93243924ff1f76488ba94cf8/app/views/admin/editions/show.html.erb#L5) so I've kept it wrapped around similar metadata for the locked view in case it's still needed

Reply:
Sure, sounds reasonable.